### DISCUSSION TITLE
1.13.0

### DISCUSSION THREAD

**Discussion by th-ch** [pear-devs/pear-desktop#408] [General]
- update various dependencies
- improve readme - including a more precise list of plugins (by @konhi)
- use node v14
- add plugin to fetch lyrics from Genius
- MPRIS support on Linux (by @itzmanish)
- Tuna plugin for ops (by @mesmerx)
- fix rendering of menu buttons 

Credits to @Araxeus and @cpiber for their help on PR reviews +  all contributors of opened PRs (that have not been reviewed yet 🙈)

<hr /><em>This discussion was created from the release <a href='https://github.com/th-ch/youtube-music/releases/tag/v1.13.0'>1.13.0</a>.</em>

**Comment by andrealine** [comment]
How To Switch To Audio Mode ?
